MAINTAINERS.yml

```diff
@@ -1990,6 +1990,22 @@ Networking:
     - "area: Networking"
     - "area: OpenThread"
 
+"Networking: Wi-Fi":
+  status: maintained
+  maintainers:
+    - jukkar
+  collaborators:
+    - rlubos
+  files:
+    - doc/connectivity/networking/api/wifi.rst
+    - include/zephyr/net/wifi*.h
+    - subsys/net/l2/wifi/
+    - samples/net/wifi/
+    - tests/net/wifi/
```

Comment:
you probably want to exclude those from the networking area

Reply:
I've added those entries to `files-exclude:` in Networking section.